data/data.json

```
@@ -143,19 +143,19 @@
         },
         "main_summary": {
             "attr": "検査実施人数",
-            "value": 338,
+            "value": 585,
```

```suggestion
            "value": 733,
```
検査実施人数ですが、現時点では733人となっています。
https://www.pref.miyazaki.lg.jp/kansensho-taisaku/kenko/hoken/covid19.html

---

失礼しました！修正しました